lib/WebpackConfig.js

```diff
@@ -251,11 +251,17 @@ class WebpackConfig {
         this.sassLoaderOptionsCallback = sassLoaderOptionsCallback;
 
         for (const optionKey of Object.keys(options)) {
-            if (!(optionKey in this.sassOptions)) {
-                throw new Error(`Invalid option "${optionKey}" passed to enableSassLoader(). Valid keys are ${Object.keys(this.sassOptions).join(', ')}`);
+            let normalizedOptionKey = optionKey;
+            if (optionKey === 'resolve_url_loader') {
+                logger.warning('enableSassLoader: "resolve_url_loader" is deprecated. Please use "resolveUrlLoader" instead.');
```

Comment:
How about a new `logger.deprecation` method? It will probably look the same, but can be a nice, standard way to start doing this

Reply:
👍